src/log/dispatch.cr

```diff
@@ -0,0 +1,91 @@
+class Log
+  # Base interface implemented by log entry dispatchers
+  #
+  # Dispatchers are in charge of sending log entries according
+  # to different strategies.
+  module Dispatcher
+    alias Spec = Dispatcher | DispatchMode
```

Comment:
What is this alias?

Reply:
Is used in the `IOBackend` [initializer](https://github.com/crystal-lang/crystal/blob/65729d9d51a01bf9b8148a2a0f5113d129cb25b6/src/log/io_backend.cr#L6) as a shorthand. It defines what can be specified for the dispatcher in the base class `Backend`.